src/lib.rs

```diff
@@ -53,28 +77,15 @@ mod bridge_state;
 #[cfg(test)]
 mod testing;
 
+#[cfg(feature = "std")]
 use std::fmt;
```

Comment:
we can probably remove this import and just refer to `std::fmt` where this is used below.